vector/src/actcwd/google-services.json

```diff
@@ -0,0 +1,48 @@
+{
+  "project_info": {
+    "project_number": "345143529889",
+    "firebase_url": "https://act-riot-app.firebaseio.com",
+    "project_id": "act-riot-app",
+    "storage_bucket": "act-riot-app.appspot.com"
+  },
+  "client": [
```

Comment:
Should this info be public or doesn't it matter.

Reply:
Rafi, can you please change act-riot-app references to pegacorn-communicate-app

Reply:
This is completely generated by a firebase account. I just created that file from my own account to run the app, otherwise, it will have build error.